src/functions.js

```diff
@@ -487,6 +585,14 @@ var _ = Mavo.Functions = {
 
 var $u = _.util;
 
+// Converts all unary Math functions that haven't been defined 
+// in Mavo.Functions to accept an array argument.
+for (const property of Object.getOwnPropertyNames(Math)) {
```

Comment:
Avoid using `for..of` loops for now. They are lovely, but when transpiled by Babel they produce a bloated mess. See: http://babeljs.io/repl#?babili=false&browsers=last%204%20versions%2C%20Chrome%20%3E%3D%2041%2C%20IE%2011&build=&builtIns=false&spec=false&loose=false&code_lz=GYewTgBAFAbghpAlhEwILASggbwFAQR4C-QA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&sourceType=module&lineWrap=false&presets=stage-2%2Cenv&prettier=false&targets=&version=6.26.0&envVersion=1.6.2

Use `array.forEach` instead.